vagrant/provisioning/install.sh

```diff
@@ -1,5 +1,11 @@
 sudo apt-get update
-sudo apt-get install -y python3-pip perl
-sudo pip3 install pipenv
+sudo apt-get install -y python3-pip perl python3-venv
 sudo curl -L https://get.docker.com/ | bash
 gpasswd -a vagrant docker
+curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/install-poetry.py | python3 -
+echo __________________________________
+id
```

Comment:
Is this a command? Or should also be in echo?

Reply:
id is actually a command which shows you your user and groups, it is for debugging purposes in there.